components/dashboard/src/components/DropDown2.tsx

```diff
@@ -140,7 +140,7 @@ export const DropDown2: FunctionComponent<DropDown2Props> = (props) => {
                 className={
                     "h-16 bg-gray-100 dark:bg-gray-800 flex items-center px-2 " +
                     (showDropDown
-                        ? "rounded-t-lg"
+                        ? "rounded-t-lg filter drop-shadow-xl"
```

Comment:
Pulled in changes started in https://github.com/gitpod-io/gitpod/pull/17680 since I was in here already.